build/buildRunTimeImageBases.sh

```diff
@@ -45,6 +45,13 @@ echo
 # Build the common base image first, so other images that depend on it get the latest version.
 # We don't retrieve this image from a repository but rather build locally to make sure we get
 # the latest version of its own base image.
+docker build \
```

Comment:
This change seems to impact more than just this PR. Were we not building the right runtime images anymore?
Also, should we also update in this file the `if [ "$runtimeSubDir" == "node" ]; then` block with the buster docker build, in this case?

Reply:
Yes we found the baseImages pipelines failed since Jun, here we put back docker build buster based runtime images so that the ACR images will get built in the pipeline. Node contains only bullseye based images now, so we won't need to put one under there.

Reply:
Thanks for clarifying!